src/main/java/com/flightstats/hub/dao/aws/S3Config.java

```diff
@@ -17,15 +17,21 @@
 import com.google.common.util.concurrent.AbstractScheduledService;
 import com.google.inject.Inject;
 import com.google.inject.name.Named;
+import lombok.extern.slf4j.Slf4j;
 import org.joda.time.DateTime;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
-import java.util.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.Random;
+import java.util.SortedSet;
+import java.util.TreeSet;
 import java.util.concurrent.TimeUnit;
 
+@Slf4j
 public class S3Config {
-    private final static Logger logger = LoggerFactory.getLogger(S3Config.class);
+
+    private static final Integer S3_LIFECYCLE_RULES_MAX = 1000;
```

Comment:
Might I suggest you set this to 990 or 999 so that by default it doesn't interfere with the terraform rule we'd like to have? I suppose if we use one of those, we'd either need to change the name or add an explanatory comment, though.

Reply:
I thought about it. I was not sure number of rules we will adding  in terraform. Given the scarcity of rules I did not want to pick a number :)